### DISCUSSION TITLE
Docker Installation on Single Node

### DISCUSSION THREAD

**Discussion by michallawson** [HCL-TECH-SOFTWARE/connections-automation#49] [Q&A] [answered]
When I try and install the component pack (single node) I receive an error in TASK [docker-install :  Add {{__kubectl_user}} to the group docker] The task includes an option with an undefined variable. The error was : {{kubectl_user  | default (ansible_env['SUDO_USER']) dict object has no attribute 'SUDO_USER' 

Will be grateful for you advice on what is wrong.

Many thanks

Michael Lawson

**Comment by dejo1307** [comment, marked as answer]
Hey Michael, what OS you are running it on, and what version? 

Normally, (ansible_env['SUDO_USER']) means that whatever user is running Ansible is registered inside the machine in Ansible environment as SUDO_USER. 

Also, you can add to the bottom of your inventory file this line:

kubectl_user="michallawson"

...and in case your user that you are running Ansible with is that one, it will set up everything needed for that user. Can you try that?

**Reply by michallawson** [reply]
Thank you for your help.

Following the earlier problems, I gave up on Centos 8 and reverted to Centos 7. Everything is now running for the installation of DB2, TD, WAS & Connections but think I may have made a mistake by setting the remote_user variable to root as WAS will now only run as the root user.  

Will tweak the remote_user setting to a SUDO user and run the installation of connections and the component pack to see if this resolves the problem. 

Kind regards

Michael

**Reply by dejo1307** [reply]
Regarding CentOS 8 - we are looking into that topic currently, but that is a part of a bigger discussion regarding the builds as well, not only setting up third part software. My feeling currently is - from what I saw with both RHEL and CentOS - is to stick with v7 and run yum update. At least for the moment being.

**Reply by michallawson** [reply]
I do not envy you and I will be sticking to Centos 7 for some time.

Changed the remote_user variable to SysAdmin and installation seems to have completed and just need to test everything.
